app/templates/services/edit_submission_section.html

```diff
@@ -35,32 +35,26 @@
 
 {% block save_button %}
 
-  {% if next_section_name and not force_return_to_summary %}
+  {% if force_continue_button or (next_question and not force_return_to_summary) %}
     {%
       with
       label="Save and continue",
-      name = "continue_to_next_section",
+      name="save_and_continue",
       type="save"
     %}
       {% include "toolkit/button.html" %}
     {% endwith %}
 
-    <p class="next-page-message">
-      Next: {{ next_section_name }}
-    </p>
-
-    {%
-    with
-      type = "secondary",
-      label = "Save and return to service overview"
```

Comment:
Going to leave this as-is and will review with @ralph-hawkins and @cathrooney when it's on preview.